.remarkrc.js

```diff
@@ -8,7 +8,6 @@ exports.plugins = [
   require('remark-preset-lint-markdown-style-guide'),
   ['lint-blockquote-indentation', false],
   ['lint-emphasis-marker', false],
-  // ['lint-fenced-code-flag', false],
```

Comment:
Do you want to remove this? I think it grumps when people forget to include a language with their code fence.

Reply:
Yes, I'd like it to grump. The default is true, so leaving it out is good.